cmd/webhook/main.go

```diff
@@ -104,6 +104,9 @@ func withCheckDefaultStorageClass(ctx context.Context, storageClassLister lister
 
 		if val, ok := sc.Annotations["storageclass.kubernetes.io/is-default-class"]; ok && val == "true" {
 			ctx = context.WithValue(ctx, buildapi.HasDefaultStorageClass, true)
+			if val, ok := sc.Annotations["storageclass.kubernetes.io/is-immutable"]; ok && val == "true" {
```

Comment:
Happy to change this annotation to whatever others feel best. Maybe kpack io domain might be preferable.

Reply:
Maybe we could check `sc.AllowVolumeExpansion` instead of inventing an annotation. See https://kubernetes.io/docs/concepts/storage/storage-classes/#allow-volume-expansion

Reply:
@phil9909 thanks for the suggestion! much cleaner!